cajita/src/Cajita_Halo.hpp

```diff
@@ -14,7 +14,8 @@
 
 #include <Cajita_Array.hpp>
 #include <Cajita_IndexSpace.hpp>
-#include <Cajita_ParameterPack.hpp>
+
+#include <Cabana_Core.hpp>
```

Comment:
Any reason to include this and not just `<Cabana_ParameterPack.hpp>`?

Reply:
Done